lib/quantum/job_broadcaster.ex

```diff
@@ -34,8 +34,10 @@ defmodule Quantum.JobBroadcaster do
 
   @doc false
```

Comment:
I think your merge was not clean. What changes did you make here?

Reply:
Yeah I messed up a couple of times doing the rebase stuff...
I think I fixed that in the latest commit.
Still TravisCI still complains about the formatting issues.
Though when I try to do `mix format` a while bunch of changes pops up.
I believe I should only commit format changes to the files Travis is complaining about (there will be three of them).
I'll try do to that shortly.